website/docs/r/ovh_dedicated_server_networking.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "ovh"
+page_title: "OVH: ovh_dedicated_server_networking"
+sidebar_current: "docs-ovh-resource-ovh-dedicated-server-networking"
+description: |-
+  Manage dedicated server networking interface
+---
+
+# ovh_dedicated_server_networking
+
+Manage dedicated server networking interface.
+
+## Example Usage
+
+The following example aims to bond all interfaces in vRack.
+
+```hcl
+locals {
+  dedicated_server = "nsXXXXXX.ip-XX-XXX-XX.eu"
+}
+
+data "ovh_dedicated_server" "server" {
+  service_name = local.dedicated_server
+}
+
+resource "ovh_dedicated_server_networking" "server"{
+  service_name = local.dedicated_server
+  interfaces {
+      macs = sort(flatten(data.ovh_dedicated_server.server.vnis.*.nics)),
+      type = "vrack"
+  }
+} 
+```
+
+The following example aims to attach the server to two different vRack.
+
+```hcl
+locals {
+  dedicated_server = "nsXXXXXX.ip-XX-XXX-XX.eu"
+}
+
+data "ovh_dedicated_server" "server" {
+  service_name = local.dedicated_server
+}
+
+resource "ovh_dedicated_server_networking" "server"{
```

Comment:
I have an error when trying to execute this example:

```
Terraform will perform the following actions:

  # ovh_dedicated_server_networking.server will be created
  + resource "ovh_dedicated_server_networking" "server" {
      + description  = (known after apply)
      + id           = (known after apply)
      + service_name = "ns<censored>.ip-xx-xx-xx.eu"
      + status       = (known after apply)

      + interfaces {
          + macs = [
              + "d0:xx:xx:xx:xx:b3",
              + "d0:xx:xx:xx:xx:b4",
            ]
          + type = "vrack"
        }
    }
    
 ...
    
ovh_dedicated_server_networking.server: Creating...
ovh_dedicated_server_networking.server: Still creating... [10s elapsed]
╷
│ Error: Error calling POST /dedicated/server/nsxxxxx.ip-xx-xx-xx.eu/networking:
│ 	 "HTTP Error 403: \"This account is not eligible to this API call\""
│
│   with ovh_dedicated_server_networking.server,
│   on pr351.tf line 11, in resource "ovh_dedicated_server_networking" "server":
│   11: resource "ovh_dedicated_server_networking" "server"{
```

Do you know why?

Thanks

Reply:
This is expected to be able to use it, for the moment, your nic must be allowed by baremetal team.
ping me when you want to test it as I have the necessary to test it with you.